src/OrbitGl/TimeGraph.cpp

```diff
@@ -74,7 +74,7 @@ void TimeGraph::Clear() {
   std::lock_guard<std::recursive_mutex> lock(mutex_);
 
   batcher_.StartNewFrame();
-  capture_min_timestamp_ = std::numeric_limits<uint64_t>::max();
+  capture_min_timestamp_ = 0;
```

Comment:
Does it mean that `capture_min_timestamp_ ` won't be updated after calling this function?

Reply:
Ok, I see you are referring to UpdateCaptureMinMax. I will revert this part for now, This change was made because by default capture_min_timestamp_ is 0 when class is constructed. I will look into this separately.

Reply:
ok, thank you!

Reply:
ok, without this change we have a crash. taking another look.